src/commands/moderation/ban.ts

```diff
@@ -129,7 +129,7 @@ const noDM = (user: User) => makeEmbed({
 
 export const ban: CommandDefinition = {
     name: 'ban',
-    requiredPermissions: ['BanMembers'],
+    requirements: { permissions: ['BanMembers'] },
```

Comment:
Not a comment on the PR, but @benw202 : Does this make sense, should we take the opportunity to change this to ADMIN & MODERATOR roles?

This is, perhaps with unban, the only one that makes a bit of sense :)